javacs-lab12/src/com/flatironschool/javacs/WikiSearch.java

```diff
@@ -159,13 +176,83 @@ public static WikiSearch search(String term, JedisIndex index) {
 		return new WikiSearch(map);
 	}
 
+	public static WikiSearch searchTFIDF(String term, JedisIndex index) {
+		if(index.getCountsTFIDF(term) == null) { return null; }
+		Map<String, Integer> map = index.getCountsTFIDF(term);
```

Comment:
same thing about calling the same method twice. except now calling this twice ends up calling getTFIDF 4 times and all that